DockerfileSlave

```diff
@@ -4,7 +4,7 @@ FROM jenkinsci/jnlp-slave:3.35-2-jdk11
 
 USER root
 
-RUN apt-get update \
+RUN apt-get update || apt-get update \
```

Comment:
Why running update 2 time?

Reply:
Not sure... The reason is:
* actual code, you try to build the slave, it fails on that step (forgot exactly the error). Tried to check a bit, could not solve it for the moment
* this is what I found in the repo on `james-charge-02`, the file was modified but not committed, and with it it builds

Reply:
Uncommited changes to bypass a review is evil. 

Ok for merging this "hotfix" and not investing more time into that...

Reply:
yes I found this workaround somewhere, don't remind exactly why but it did not work without it... I probably did not commit this because it was too ugly for me...

Reply:
Ahah alright, still I almost lost that workaround so better to push it, even if ugly :)